src/Domain/Metrics/Architecture/Classes.php

```diff
@@ -39,6 +40,7 @@ public function getInsights(): array
             OneClassPerFileSniff::class,
             SuperfluousInterfaceNamingSniff::class,
             SuperfluousAbstractClassNamingSniff::class,
+            ForbiddenFinalClasses::class,
```

Comment:
:thinking: I think we shouldn't add it here, this will cause forbiddenFinalClasses for everyone